st2common/st2common/util/action_db.py

```diff
@@ -195,6 +196,11 @@ def update_liveaction_status(status=None, result=None, context=None, end_timesta
     old_status = liveaction_db.status
     liveaction_db.status = status
 
+    liveaction_db.log.append({
```

Comment:
Checkout https://github.com/StackStorm/st2/blob/1df8f4cdd4fc4e3bd1eac79732d1b3aac1be4745/st2common/st2common/persistence/trace.py#L28

Idea is to use the push operator. Likely applies here as well.